wdl2cwl/tests/wdl_files/hisat2_1.wdl

```diff
@@ -59,7 +60,7 @@ task Hisat2 {
         hisat2 \
         -p ~{threads} \
         -x ~{indexFiles[0]} \
-        ~{true="-1" false="-U" defined(inputR2)} ~{inputR1} \
+        ~{true="-1" false="-U" inputR2} ~{inputR1} \
```

Comment:
Is there an issue for our lack of support for WDL's `defined()`?

Reply:
No. I created a new issue